packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -40,7 +40,8 @@ export type CallCompositePage =
   | 'lobby'
   | /* @conditional-compile-remove(rooms) */ 'deniedPermissionToRoom'
   | 'removedFromCall'
-  | /* @conditional-compile-remove(rooms) */ 'roomNotFound';
+  | /* @conditional-compile-remove(rooms) */ 'roomNotFound'
+  | /* @conditional-compile-remove(unsupported-browser) */ 'unsupportedBrowser';
```

Comment:
per other comment if we decide to so that, update from unsupportedBrowser to unsupportedEnvironment

Reply:
Which other comment? did you want me to change the feature flag here after our discussion this morning?

Reply:
![image](https://user-images.githubusercontent.com/2684369/194585842-6bf883bd-c483-4989-a818-80b8daaceb37.png)
^this one